lib/Sema/CSFix.cpp

```diff
@@ -947,6 +947,21 @@ bool AllowMemberRefOnExistential::diagnose(const Solution &solution,
   return failure.diagnose(asNote);
 }
 
+bool AllowInvalidMemberRef::diagnoseForAmbiguity(
+    CommonFixesArray commonFixes) const {
+  auto *primaryFix =
+      static_cast<const AllowInvalidMemberRef *>(commonFixes.front().second);
```

Comment:
It would result in the same, but I remember we recently introduced a `castTo` in `ConstraintFix`. Would it be applicable here?

Reply:
Cannot use that for `AllowInvalidMemberRef` because it's a base class for multiple fixes.